lib/std/os/bits/linux.zig

```diff
@@ -764,19 +764,14 @@ pub const winsize = extern struct {
     ws_ypixel: u16,
 };
 
+/// NSIG is the total number of signals defined.
+/// As signal numbers are sequential, NSIG is one greater than the largest defined signal number.
 pub const NSIG = if (is_mips) 128 else 65;
-pub const sigset_t = [128 / @sizeOf(usize)]usize;
```

Comment:
I considered making this a `std.PackedIntArray(u1, 1024)`. Let me know if you'd like that.